analytics/tests/integrations/github/test_validation.py

```diff
@@ -42,6 +43,36 @@
 }
 
 
+class TestTransformProjectData:
```

Comment:
Nice test! Clean and simple.

Would it be worth adding a line like this to the snapshot test as well?